src/coverlet.core/Abstracts/IFileSystem.cs

```diff
@@ -1,7 +1,23 @@
-﻿namespace Coverlet.Core.Abstracts
+﻿using System.IO;
+
+namespace Coverlet.Core.Abstracts
 {
-    internal interface IFileSystem
+    public interface IFileSystem
```

Comment:
we shouldn't expose internal interface if you need to expose to console update https://github.com/tonerdo/coverlet/blob/master/src/coverlet.core/Properties/AssemblyInfo.cs

Reply:
I need to expose it to console and msbuild.task.

Reply:
You're right we did the same for instrumentation helper, maybe in future we should sign and move to internal, never mind